### PR TITLE
MiQ-Bot now with added update and setup

### PR DESCRIPTION
**Pull request by juliancheal**
For consistancy and ease of use basically a copy of MangeIQ's
setup and update commands.

Also the beginings of following along to https://github.com/github/scripts-to-rule-them-all

Fixes issue #175

### CODE REVIEW

File: bin/setup [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+  puts "\n== Copying sample files =="
+  unless File.exist?("config/database.yml")
+    system "cp config/database.tmpl.yml config/database.yml"
+  end
+  unless File.exist?("Procfile")
+    system "cp Procfile.tmpl Procfile"
+  end
+  unless File.exist?("config/settings/development.local.yml")
+    system "cp config/settings.yml config/settings/development.local.yml"
```
Comment (bdunne) [inline bin/setup]: Why create this file if it contains no changes?

File: bin/update [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env ruby
+require 'pathname'
+
+# path to your application root.
+APP_ROOT = Pathname.new File.expand_path('../../',  __FILE__)
+
+Dir.chdir APP_ROOT do
+  puts "== Installing dependencies =="
+  system "bundle update"
+  system "bower update --allow-root -F --config.analytics=false"
```
Comment (bdunne) [inline bin/update]: I don't think we use bower in this app.

Comment (juliancheal) [inline bin/update]: Cool, I'll remove that. :)

File: bin/setup (hunk 1 above)
Comment (juliancheal) [inline bin/setup]: Sorry, I'm not sure what you mean?

Comment (bdunne) [inline bin/setup]: I don't think it makes sense to copy `config/settings.yml`.  I would only expect that file to contain intentional differences.

File: bin/update (hunk 2 above)
Comment (himdel) [inline bin/update]: ..but we probably *will* need npm :) (JS linters..)

**Pull request closed without merging**